app/app.go

```diff
@@ -18,24 +19,38 @@ import (
 )
 
 var (
-	config  *conf.Config
 	chain   *blockchain.BlockChain
 	logFile = os.Stdout
 	// A reference to the transaction pool
 	tpool *pool.Pool
 )
 
+// App contains information about a running instance of a Cumulus node
+type App struct {
```

Comment:
Yea, I guess we could. It's not really used in other packages anyway. What do you think @jordanschalm?